src/plugins/intel_cpu/src/graph_optimizer.cpp

```diff
@@ -1556,8 +1579,11 @@ void GraphOptimizer::FuseConvolutionAndSimpleOperation(Graph &graph) {
 //Disable ACL post-ops in fp16 to avoid performance degradation
 #if defined(OPENVINO_ARCH_ARM64)
         if (parentNode->getOriginalInputPrecisionAtPort(0) == ov::element::f16) {
-            parent++;
-            continue;
+            bool isFirstSupportedPostOp = parentNode->getFusedWith().empty() && is_post_op_supported(childNode);
+            if (!isFirstSupportedPostOp) {
+                parent++;
+                continue;
+            }
         }
 #endif
```

Comment:
1. Lets remove untire code block, parentNode->canFuse(childNode) should handle the check correctly. 
2. Convolution::canFuse should contain `getFusedWith().empty()` check under OV_CPU_WITH_ACL  macro

Reply:
fixed